packages/node_modules/node-red/settings.js

```diff
@@ -206,131 +194,109 @@ module.exports = {
-
-    // `global.keys()` returns a list of all properties set in global context.
-    // This allows them to be displayed in the Context Sidebar within the editor.
-    // In some circumstances it is not desirable to expose them to the editor. The
-    // following property can be used to hide any property set in `functionGlobalContext`
-    // from being list by `global.keys()`.
-    // By default, the property is set to false to avoid accidental exposure of
-    // their values. Setting this to true will cause the keys to be listed.
-    exportGlobalContextKeys: false,
-
-    // Uncomment the following to run node-red in your preferred language:
-    // lang: "de",
-
-    // Context Storage
-    // The following property can be used to enable context storage. The configuration
-    // provided here will enable file-based context that flushes to disk every 30 seconds.
-    // Refer to the documentation for further options: https://nodered.org/docs/api/context/
-    //
-    //contextStorage: {
-    //    default: {
-    //        module:"localfilesystem"
-    //    },
-    //},
-
-    // The following property can be used to order the categories in the editor
-    // palette. If a node's category is not in the list, the category will get
-    // added to the end of the palette.
-    // If not set, the following default order is used:
-    //paletteCategories: ['subflows', 'common', 'function', 'network', 'sequence', 'parser', 'storage'],
-
-    // Configure the logging output
-    logging: {
-        // Only console logging is currently supported
-        console: {
-            // Level of logging to be recorded. Options are:
-            // fatal - only those errors which make the application unusable should be recorded
-            // error - record errors which are deemed fatal for a particular request + fatal errors
-            // warn - record problems which are non fatal + errors + fatal errors
-            // info - record information about the general running of the application + warn + error + fatal errors
-            // debug - record information which is more verbose than info + info + warn + error + fatal errors
-            // trace - record very detailed logging + debug + info + warn + error + fatal errors
-            // off - turn off all logging (doesn't affect metrics or audit)
-            level: "info",
-            // Whether or not to include metric events in the log output
-            metrics: false,
-            // Whether or not to include audit events in the log output
-            audit: false
-        }
-    },
+/*******************************************************************************
+ * Runtime Settings
+ *  - lang
+ *  - logging
+ *  - contextStorage
+ *  - exportGlobalContextKeys
+ *  - externalModules
+ ******************************************************************************/
+
+     // Uncomment the following to run node-red in your preferred language:
+     // lang: "de",
```

Comment:
should we mention which languages we have translations for ? (and maybe encourage contributions... ?)